tests/convert_utf32_to_utf16be_with_errors_tests.cpp

```diff
@@ -15,6 +15,30 @@ using simdutf::tests::helpers::transcode_utf32_to_utf16_test_base;
 constexpr int trials = 1000;
 } // namespace
 
+TEST(issue_convert_utf32_to_utf16be_with_errors_fb5c30a7d5815504) {
```

Comment:
maybe rename this to something that makes more sense to what the actual testcase is?